app/helpers/mocks.rb

```diff
@@ -12,6 +12,39 @@ def self.pii_mock(user_ids)
     user_ids.map { |u| { 'id' => u, 'dob' => '1993-01-01', 'email' => "#{u}@worldcubeassociation.org" } }
   end
 
+  def self.user_info_mock(user_ids)
+    iso = %w[AD AE AI AL BW BY BZ CA CC NU NZ OM PA PE PN PR PS PTF TG TH TJ WS ZW].sample
+    {
+      'users' => user_ids.map do |u|
+        {
+          'id' => u,
+          'created_at' => Time.now.to_s,
+          'updated_at' => Time.now.to_s,
+          'name' => "Name #{u}",
+          'wca_id' => "2023TEST#{u % 99}",
+          'delegate_status' => nil,
+          'gender' => 'm',
+          'country_iso2' => iso,
+          'url' => "https://#{EnvConfig.WCA_HOST}/persons/2023TEST#{u % 99}",
```

Comment:
Why are you not refactoring the WCA ID into a variable if it's used in two places?